app/views/notifications/_form.html.erb

```diff
@@ -2,10 +2,11 @@
   <h3><%= "#{@event.title} [#{@event.id_code}] "  %></h3>
   <h4>(<%= format_time_span(@event.start_time, @event.end_time, @event.duration) %>)</h4>
   <%= f.error_notification %>
+  <div class="alert alert-danger"><%= f.error :base %></div>
 
   <div class="form-inputs">
     <%= f.association :event,
-                      collection: Event.upcoming,
+                      collection: Event.upcoming + [@event],
```

Comment:
Could we have `upcoming` include this event?

Reply:
No, because we may be sending out a notification about an event in the past. ("Be sure to have all your paperwork regarding Hurricane Harvey turned in by next Friday")